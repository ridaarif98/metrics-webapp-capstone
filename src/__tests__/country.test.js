import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import CountryDislay from '../compoenets/country';

describe('Test to see that all objects render correctly.', () => {
  test('Check if Continent renders.', () => {
    const tree = renderer
      .create(
        <Router>
          <Provider store={store}>
            <CountryDislay />
          </Provider>
        </Router>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
