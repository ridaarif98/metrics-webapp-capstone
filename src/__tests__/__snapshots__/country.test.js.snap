// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to see that all objects render correctly. Check if Continent renders. 1`] = `
<div
  className="countryData"
>
  <div
    className="countryHeader"
  >
    <img
      alt="Europe"
      src="europe.png"
    />
    <div
      className="heading"
    >
      <h2 />
      <span />
    </div>
  </div>
  <h1
    className="cityHeading"
  >
    CITY/ TOWN BREAKDOWN
  </h1>
  <ul
    className="countryDetail"
  />
</div>
`;
